website/static/js/addProjectPlugin.js

```diff
@@ -87,6 +87,15 @@ var AddProject = {
         };
     },
     view : function (ctrl, options) {
+        var capitalizeCategories = function(category) {
```

Comment:
We have a mapping of internal names to human-readable names here: https://github.com/CenterForOpenScience/osf.io/blob/54418bc5fbce98fdb892267a27a93a560b06433a/website/project/model.py#L707-L719 . 

I'd rather see a mapping like this used (even if it means duplicating it in this JS file) than this capitalize function.

The internal names (e.g. `materials and methods`) are meant to be stable; the human-friendly names may be subject to change (e.g. we might decide to display `Methodology/Materials`). Using a mapping allows us to make this change easily, and prevents unexpected bugs when we add new categories.